ptbcontrib/redis_persistence/README.md

```diff
@@ -0,0 +1,37 @@
+# Bot persistence via a Redis database
+## Notes
+
+    1. `RedisPersistence` class only writes data to the database when the bot is shut down.
+    2. As `RedisPersistence` is a subclass of `DictPersistence`, all data set to be stored by the class must be JSON-serializable.
+
+## How to use
+### Passing URL of the database
+```python
+import telegram
+from ptbcontrib.redispersistence import RedisPersistence
+
+persistence = RedisPersistence(url = 'redis://username:password@localhost:12345')
+
+updater = telegram.ext.Updater(..., persistence = persistence, ...)
```

Comment:
Putting spaces in function parameters is not correct according to PEP 8. Since those are code examples, having them up to standard would be great.